UI/chips.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>fast-food</title>
+	<link rel="stylesheet" type="text/css" href="assets/css/main.css">
+	<script type="text/javascript" src="assets/js/main.js"></script>
+</head>
+<body>
+	<div class="header">
+		<div class="left">
+			FAST-FOOD
+		</div>
+
+		<div class="right">
+			<ul>
+					<li><a href="foodmenu.html">Make orders</a></li>
+					<li class="active"><a href="foodhistory.html">Order History</a></li>
+					<li><a href="../index.html">Logout</a></li>
+			</ul>
+		</div>
+		
+	</div>
+
+	<div class="content">
+		<div class="info">
+			<img src="assets/images/food.jpg">
+			
+			<div class="content-infomation">
+				<button class="menu">Chips</button>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<button class="menu">Price</button>
```

Comment:
It would be more readable, and consistent  if you used a constant  padding around these elements than spaces(`&nbsp;`) around them